core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1892,7 +1892,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
     }
     val callSite = getCallSite
     val cleanedFunc = clean(func)
-    logInfo("Starting job: " + callSite.shortForm)
+    logDebug("Starting job: " + callSite.shortForm)
```

Comment:
@hbhanawat I think we need a way to turn on these task logging if required without having to go to debug level (which will be too much), rather more log4j way. How about adding a separate "taskLogger" which uses its own className in slf4j logger initialization so that it can be tuned separately from log4j.properties

```
@transient val taskLogger = LoggerFactory.getLogger(logName + ".Task")
...
taskLogger.logInfo(...)
```

Then the default for "org.apache.spark.SparkContext.Task" can be changed in store-log4j.properties to be WARN.